shipwrighttests/features/02_generate_new_image.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Build container images for nodejs app using "buildpacks-v3" and "clusterbuildstrategies"
+ 
+ With Shipwright, developers get a simplified approach for building container images, by defining
+ a minimal YAML that does not require any previous knowledge of containers or container tooling. 
+ All you need is your source code in git and access to a container registry.
+ 
+ Shipwright supports any tool that can build container images in Kubernetes clusters, such as:
```

Comment:
I think the "feature" block should focus on the feature itself, so instead of what Shipwright does, we should focus on explaining what is expected from the given feature and what the testing entails.

Reply:
Made the changes